tools/upload_text.php

```diff
@@ -293,6 +303,7 @@
     if (strlen($postcomments)>0 || $have_file) {
         $postcomments = $divider . $postcomments;
     }
+    // note that $postcomments is used as a global variable inside do_state_change() inside project_transition()
```

Comment:
Good comment - thank you for adding it.

Reply:
It took me a while to figure out what was happening to postcomments